app/controllers/admin/booths_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class Admin::BoothsController < Admin::BaseController
+    load_and_authorize_resource :conference, find_by: :short_title
+    load_and_authorize_resource :booth, through: :conference
```

Comment:
Since you are already in the booths controller, you don't need to specify which resource you are loading/authorizing